packages/chat-ui/src/components/ChatMessages.tsx

```diff
@@ -67,7 +67,7 @@ export const ChatMessages = ({
   );
 
   const loaders = useMemo(() => {
-    return isLoading || !messages.length ? (
+    return isLoading ? (
```

Comment:
This was causing the loader to always be there when there were no messages in the chat yet.